modules/test-junit4/pom.xml

```diff
@@ -0,0 +1,35 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.devonfw.java.dev</groupId>
+    <artifactId>devon4j-modules</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
+  <groupId>com.devonfw.java.modules</groupId>
+  <artifactId>devon4j-test-junit4</artifactId>
+  <version>${devon4j.version}</version>
+  <name>${project.artifactId}</name>
+  <description>Module with code and configuration for tests.</description>
```

Comment:
could you update the description to include this is for junit4 backward compatibility.